src/API/WesternStatesWater.WestDaat.Accessors/OrganizationAccessor.cs

```diff
@@ -143,5 +144,22 @@ private async Task<OrganizationMemberRemoveResponse> RemoveOrganizationMember(Or
 
             return new OrganizationMemberRemoveResponse();
         }
+        
+        private async Task<OrganizationMemberUpdateResponse> UpdateOrganizationMember(OrganizationMemberUpdateRequest request)
+        {
+            await using var db = _westDaatDatabaseContextFactory.Create();
+
+            var userOrganization = await db.UserOrganizations
+                .Include(uo => uo.UserOrganizationRoles)
+                .FirstOrDefaultAsync(uo => uo.UserId == request.UserId && uo.OrganizationId == request.OrganizationId);
+
+            var userOrganizationRole = userOrganization.UserOrganizationRoles.First();
```

Comment:
Let's make it blow up here. This is so that if we ever move to a multi-role world it will throw an exception instead of updating the first role it finds
```suggestion
            var userOrganizationRole = userOrganization.UserOrganizationRoles.Single();
```